tests/users/test_sync_group_and_perms.py

```diff
@@ -23,4 +23,4 @@ def test_readonly_group():
     readonly_groups = {name for name in permissions.keys() if name.endswith("-readonly")}
     for group in readonly_groups:
         assert set(permissions[group].keys()) == set(permissions[group.replace("-readonly", "")].keys())
-        assert set(itertools.chain(*permissions[group].values())) == {"view"}
+        assert set(itertools.chain(*permissions[group].values())) <= sync_group_and_perms.PERMS_READONLY
```

Comment:
Vu qu'il ne me semble pas qu'on le vérifie ailleurs, ici il faut plutôt garder la duplication de code afin qu'en cas de modification de `PERMS_READONLY` le test échoue aussi ici afin de se poser la question (dev et relecteurs) de si on veux vraiment faire ça :).
```suggestion
        assert set(itertools.chain(*permissions[group].values())) <= {"view", "export_cta"}
```